src/main/java/seedu/address/logic/commands/ViewAllCommand.java

```diff
@@ -1,25 +1,25 @@
 package seedu.address.logic.commands;
 
 import static java.util.Objects.requireNonNull;
-import static seedu.address.model.Model.PREDICATE_SHOW_ALL_PERSONS;
+import static seedu.address.model.Model.PREDICATE_SHOW_ALL_RIDES;
 
 import seedu.address.logic.CommandHistory;
 import seedu.address.model.Model;
 
 /**
  * Lists all persons in the address book to the user.
  */
-public class ListCommand extends Command {
+public class ViewAllCommand extends Command {
 
-    public static final String COMMAND_WORD = "list";
+    public static final String COMMAND_WORD = "view all";
```

Comment:
Use a single word command, eg "viewall". Current implementation of parser does not support such a "command word".